pkg/authorization/cache/policy_cache.go

```diff
@@ -18,6 +18,18 @@ import (
 	bindingregistry "github.com/openshift/origin/pkg/authorization/registry/policybinding"
 )
 
+// PolicyCacheAdapter exposes getter methods for policy indexers and reflectors for the project cache
+type PolicyCacheAdapter interface {
```

Comment:
For each indexed item, we need to be able to list ALL and list ALL in a namespace, we could look to make that change here, but it felt like a larger change than just getting a common set of indexers.

Agree we need to doc what the requirements are for the indexer to index on (particularly namespace)